Src/Support/Google.Apis.Core/Google.Apis.Core.csproj

```diff
@@ -0,0 +1,42 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  
+  <!-- nupkg information -->
+  <Import Project="../CommonPackageInfo.xml" />
+  <PropertyGroup>
+    <Title>Google APIs Core Client Library</Title>
+    <PackageTags>Google</PackageTags>
+    <Description>
+The Google APIs Core Library contains the Google APIs HTTP layer, JSON support, Data-store, logging and so on.
+
+Supported Platforms:
+- .NET Framework 4.5+
+- NetStandard1.3
+    </Description>
+  </PropertyGroup>
+  
+  <!-- configurations; common -->
+  <PropertyGroup>
+    <TargetFrameworks>netstandard1.3;net45</TargetFrameworks>
+    <SignAssembly>true</SignAssembly>
```

Comment:
The migration tool adds

```
<PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
```

I suspect this is needed to get it to do any kind of signing on Linux. We should think about whether we want it or not.